.github/workflows/ci-build.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     timeout-minutes: 5
     steps:
       - name: Cancel previous
-        uses: styfle/cancel-workflow-action@0.10.1
+        uses: styfle/cancel-workflow-action@0.11.0
```

Comment:
I'm not sure why these changed? Revert?

Reply:
it is automatically generate github bots. I manually removed it.